source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -165,13 +167,19 @@ let parse_models_and_queries_from_sources
 let parse_models_and_queries_from_configuration
     ~scheduler
     ~static_analysis_configuration:
-      { Configuration.StaticAnalysis.verify_models; configuration = { taint_model_paths; _ }; _ }
+      {
+        Configuration.StaticAnalysis.verify_models;
+        configuration =
+          { taint_model_paths; python_major_version; python_minor_version; python_micro_version; _ };
+        _;
+      }
     ~taint_configuration
     ~resolution
     ~source_sink_filter
     ~callables
     ~stubs
   =
+  let python_version = ModelParser.PythonVersion.from_integers python_major_version python_minor_version python_micro_version in
```

Comment:
Couldn't we use `Taint.ModelParser.PythonVersion.from_configuration` here too?

Reply:
Hm, yes if we don't pattern match above, I can do this :)